ddtrace/llmobs/_constants.py

```diff
@@ -34,6 +34,7 @@
 EVP_PROXY_AGENT_ENDPOINT = "{}/api/v2/llmobs".format(EVP_PROXY_AGENT_BASE_PATH)
 EVP_SUBDOMAIN_HEADER_NAME = "X-Datadog-EVP-Subdomain"
 EVP_SUBDOMAIN_HEADER_VALUE = "llmobs-intake"
+EVP_PAYLOAD_SIZE_LIMIT = 5_000_000  # 5MB
```

Comment:
Correct me if I'm wrong (I'm not too familiar with EVP's exact limits) but shouldn't it be `5 << 20` (5 * 2^20) to get 5 MB? Or should we just stick with 5000000 (decimal system) to be safe?

Reply:
I think we can update to 5 << 20. The actual limit is 5.1MB, so this still gives us some extra space.